src/seedu/addressbook/data/person/Email.java

```diff
@@ -58,4 +59,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString() {
+        return String.format(MESSAGE_EMAIL, value);
```

Comment:
Nice try with this. You may also consider formatting strings with `Formatter`, from java.util.Formatter.